asyncapi/asgardeo/asyncapi.yml

```diff
@@ -230,6 +234,52 @@ components:
         serviceProvider:
           type: string
           example: My Account
+        authSteps:
+          type: array
+          description: A list of authentication steps
+          items:
+            $ref: '#/components/schemas/authSteps'  
```

Comment:
We have referred the schema `authSteps` here. But we have defined a different schema `authStep`.

Reply:
Resolved in [e8d6609](https://github.com/ballerina-platform/asyncapi-triggers/pull/154/commits/e8d660936bbc0a411944c843eb7e01bbc72d117c)